CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
+- To `fs.walk.Walker`, added parameters `filter_glob` and `exclude_glob`.
```

Comment:
I think "- Added `filter_glob` and `exclude_glob` parameters to `fs.walk.Walker`." sounds more natural here.